### PR TITLE
Travis: Test building of docker image and starting container

### PR DESCRIPTION
**Pull request by arjunsalyan**

### CODE REVIEW

**Comment by mojca** [comment]
There were connectivity issues during the build. We should try to re-trigger the builds in one way or another.

File: env.sample [hunk 1]
```diff
@@ -1,6 +1,6 @@
-DB_NAME=
-DB_USER=
+DB_NAME=mp
+DB_USER=postgres
 DB_PASSWORD=
-DB_HOST=
+DB_HOST=localhost
 DB_PORT=
-SECRET_KEY=
+SECRET_KEY=aRandomKey
```
Comment (mojca) [inline env.sample]: Given that you already specified some values in the Travis configuration script, could the same values be reused somehow? (It's not absolutely necessary to change it, just potentially nice.)

Other than that, it looks fine to me.

**Comment by arjunsalyan** [comment]
I restarted the build, it has passed.

File: env.sample (hunk 1 above)
Comment (arjunsalyan) [reply env.sample]: Thank you, I have done the changes.

**Review by mojca — APPROVED** [review]
Even if there is some better way to check for running image, this commit is helpful.

**Pull request merged**